Vagrantfile

```diff
@@ -29,7 +29,7 @@ end
 
 # get git config on the host so that we can have it even when running
 # ansible_local as provisioner.
-git_config = `git config --list`
+git_config = `git config --list --global`
```

Comment:
I imagine some people have system wide git config so I'd rather do :

```ruby
git_config = `git config --list --system` + "\n" + `git config --list --system`
```
If it works.

Cannot test as I am not on my computer.